org.mixedrealitytoolkit.spatialmanipulation/Solvers/RadialViewDisplayInitializeHelper.cs

```diff
@@ -0,0 +1,27 @@
+using UnityEngine;
```

Comment:
Please add change comment to "Spatial Manipulation's" CHANGELOG.md.   Following the [Keep A Changelog](https://keepachangelog.com/en/1.1.0/) pattern.

Also please change that "Spatial Manipulation's version has been properly incremented since last [release](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/releases).

Reply:
I have updated the ChangeLog but I have no idea what you mean by ""Spatial Manipulation's version has been properly incremented since last release" What am I supposed to do? Edit the asmdef?

Reply:
I think this would be updating the package.json to "version": "3.2.0-development"

Since this adds a new public class it will require a minor version bump, and the most recent release was 3.1.0. Nobody has bumped the minor version since then so this change would need to

Reply:
I think I fixed what you ask, I hope I did it correctly